eth/vm/logic/arithmetic.py

```diff
@@ -205,3 +205,18 @@ def shr(computation):
         result = (value >> shift_length) & constants.UINT_256_MAX
 
     computation.stack_push(result)
+
+
+def sar(computation):
+    """
+    Arithmetic bitwise right shift
+    """
+    shift_length, value = computation.stack_pop(num_items=2, type_hint=constants.UINT256)
```

Comment:
Side note, I just realized that we use keyword style calling for all stack operations, and a while back @davesque uncovered that calling things with positional arguments is significantly faster than keyword style calling.  We should look into updating all of these call sites to use positional arguments.